core/state_processor_rip7560.go

```diff
@@ -15,6 +14,36 @@ import (
 	"strings"
 )
 
+const MAGIC_VALUE_SENDER = uint64(0xbf45c166)
+const MAGIC_VALUE_PAYMASTER = uint64(0xe0e6183a)
+const MAGIC_VALUE_SIGFAIL = uint64(0x31665494)
+const PAYMASTER_MAX_CONTEXT_SIZE = 65536
+
+func PackValidationData(authorizerMagic uint64, validUntil, validAfter uint64) []byte {
```

Comment:
I don't see a reasonable use for `PackValidationData` outside of tests. Maybe we should move it to the test folder then?

Reply:
you're right
but it is complementary to Unpack, and I think there is a benefit on seeing (and testing) them side by side.